script.js

```diff
@@ -22,6 +54,31 @@ function clearInputs() {
 
 function deleteIdea(event) {
   if (event.target.classList.contains('delete-btn')) {
+    var id = $(event.target).parent().parent().attr('id');
+    localStorage.removeItem(id);  
     $(event.target).parent().parent().remove();
   }
+}
+
+function NewIdea(title, body) {
```

Comment:
I would recommend renaming this to `Idea`. `new Idea()` reads easier and less repetitive than `new NewIdea()`